docs/overrides.md

```diff
@@ -53,7 +53,7 @@ Just like vendored files, Linguist excludes documentation files from your projec
 Use the `linguist-documentation` attribute to mark or unmark paths as documentation:
 
 ```gitattributes
-project-docs/* linguist-documentation
+project-docs/** linguist-documentation
 docs/formatter.rb -linguist-documentation
```

Comment:
Thinking about it, lets extend the examples, rather than change them. How about:

```suggestion
# Apply override to all files in the directory
project-docs/* linguist-documentation
# Apply override to a specific file
docs/formatter.rb -linguist-documentation
# Apply override to all files and directories in the directory
ano-dir/** linguist-documentation
```